pkgs/applications/audio/spotify-cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3Packages, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
```

Comment:
It seems like this package is meant to be used as an executable rather than as a library.

If that is the case, [you should use `buildPythonApplication`](https://github.com/NixOS/nixpkgs/blob/dc1a82101ced2daf22d079d9bcc596531ab9ebac/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function) instead of `buildPythonPackage`. Additionally, the package would belong under the [`pkgs/by-name` hierarchy](https://github.com/NixOS/nixpkgs/blob/dc1a82101ced2daf22d079d9bcc596531ab9ebac/pkgs/by-name/README.md).